includes/class-wc-payments-checkout.php

```diff
@@ -415,16 +415,25 @@ function () use ( $prepared_customer_data ) {
 				);
 			}
 
-			// Output the form HTML.
-			if ( ! empty( $this->gateway->get_description() ) ) : ?>
-				<p><?php echo wp_kses_post( $this->gateway->get_description() ); ?></p>
+			?>
+			<div class="wcpay-upe-form" 
```

Comment:
This is the main wrapper, which should serve as the base for all DOM traversal moving forward. Ideally, we should aim to locate this container in the DOM and then traverse within it to find the relevant elements needed for our purposes